decko-rails/config/routes.rb

```diff
@@ -1,11 +1,7 @@
 # -*- encoding : utf-8 -*-
 
-Rails.application.routes.draw do
+Decko::Engine.routes.draw do
 
-  if !Rails.env.production? && Object.const_defined?( :JasmineRails )
```

Comment:
If I didn't miss it, this snippet should be in the generated routes.rb, maybe depending on the mode (dev vs. production).